src/Lykke.Service.Assets/Program.cs

```diff
@@ -1,15 +1,12 @@
 ﻿using System;
 using System.IO;
 using System.Threading.Tasks;
-using JetBrains.Annotations;
 using Microsoft.AspNetCore.Hosting;
 
 namespace Lykke.Service.Assets
 {
-    [UsedImplicitly]
     internal class Program
```

Comment:
Why resharper annotation is removed?

Reply:
As unnecessary attribute

Reply:
Even if Global solution analysis is enabled?